packages/terra-framework-docs/src/terra-dev-site/doc/table/Examples/TableRowSelection.jsx

```diff
@@ -0,0 +1,234 @@
+        { content: 'Inpatient, 5 months' },
+        { content: '' },
+        { content: 'Quinzell, Harleen' },
+        { content: '' },
+      ],
+    },
+    {
+      id: '5',
+      cells: [
+        { content: 'Peters, Timothy' },
+        { content: '1207-MTN' },
+        { content: 'Unstable', isMasked: true },
+        { content: 'Outpatient, 15 days' },
+        { content: '' },
+        { content: 'Quinzell, Harleen' },
+        { content: '' },
+      ],
+    },
+    {
+      id: '6',
+      cells: [
+        { content: 'Jones, Becky' },
+        { content: '1007-MTN' },
+        { content: 'Unstable' },
+        { content: 'Inpatient, 2 months' },
+        { content: '' },
+        { content: 'Quinzell, Harleen' },
+        { content: '' },
+      ],
+    },
+    {
+      id: '7',
+      cells: [
+        { content: 'Williams, Peter' },
+        { content: '1002-KTN' },
+        { content: 'Unstable' },
+        { content: 'Outpatient, 9 days' },
+        { content: '' },
+        { content: 'Quinzell, Harleen' },
+        { content: '' },
+      ],
+    },
+    {
+      id: '8',
+      cells: [
+        { content: 'Pratt, Michaela' },
+        { content: '2108-NTN' },
+        { content: 'Stable' },
+        { content: 'Outpatient, 45 days' },
+        { content: '' },
+        { content: 'Quinzell, Harleen' },
+        { content: '' },
+      ],
+    },
+    {
+      id: '9',
+      cells: [
+        { content: 'Styris, Scott' },
+        { content: '1007-MTN', isMasked: true },
+        { content: 'Unstable' },
+        { content: 'Inpatient, 2 months' },
+        { content: '' },
+        { content: 'Quinzell, Harleen' },
+        { content: '' },
+      ],
+    },
+    {
+      id: '10',
+      cells: [
+        { content: 'Cook, Allan' },
+        { content: '1700-SKB' },
+        { content: 'Stable' },
+        { content: 'Inpatient, 4 months' },
+        { content: '' },
+        { content: 'Quinzell, Harleen' },
+        { content: '' },
+      ],
+    },
+    {
+      id: '11',
+      cells: [
+        { content: 'Lahey, Nathaniel' },
+        { content: '1348-DRS' },
+        { content: 'Unstable' },
+        { content: 'Inpatient, 1 months' },
+        { content: '' },
+        { content: 'Quinzell, Harleen' },
+        { content: '' },
+      ],
+    },
+    {
+      id: '12',
+      cells: [
+        { content: 'Harris, Isabella' },
+        { content: '1809-MTN' },
+        { content: 'Stable' },
+        { content: 'Inpatient, 6 months' },
+        { content: '' },
+        { content: 'Quinzell, Harleen' },
+        { content: '' },
+      ],
+    },
+    {
+      id: '13',
+      cells: [
+        { content: 'Millstone, Asher' },
+        { content: '4133-MZN' },
+        { content: 'Unstable' },
+        { content: 'Inpatient, 9 months' },
+        { content: '' },
+        { content: 'Quinzell, Harleen' },
+        { content: '' },
+      ],
+    },
+  ],
+};
+
+const RowSelection = () => {
```

Comment:
I would have expected a test page similar to this one and some accompanying WDIO tests for row selection.

Reply:
I agree with you.  I covered in jest, but should also have added the appropriate WDIO tests.  Thank you for catching this.